demisto_sdk/commands/common/update_id_set.py

```diff
@@ -2333,6 +2511,12 @@ def re_create_id_set(id_set_path: Optional[str] = DEFAULT_ID_SET_PATH, pack_to_c
         new_ids_dict['Reports'] = []
         new_ids_dict['Widgets'] = []
         new_ids_dict['Dashboards'] = []
```

Comment:
The comment in line 2506 should only be relevant to the above object (and not the new ones below)

Reply:
Oh ok.
What is special with these objects that they need this workaround?